samples/arch/mpu/mpu_test/README.rst

```diff
@@ -1,11 +1,11 @@
 .. _mpu_test:
 
-Memory Protection Unit (MPU) TEST
+Memory Protection Unit (MPU)
```

Comment:
Maybe use `Checker` instead of `Test` (or some other more appropriate term). Removing it makes the table of content less descriptive of what this actually is.

Reply:
Addressed. MPU SAMPLE it is.